hydrator-common/pom.xml

```diff
@@ -34,6 +34,10 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>commons-lang</groupId>
```

Comment:
it is better we do not introduce this dependency, I am unsure if it has any side effect

Reply:
extract the utils method I need.